addons/common/XEH_postInit.sqf

```diff
@@ -19,11 +19,12 @@
 
 //Status Effect EHs:
 [QGVAR(setStatusEffect), {_this call FUNC(statusEffect_set)}] call CBA_fnc_addEventHandler;
-["forceWalk", false, ["ACE_SwitchUnits", "ACE_Attach", "ACE_dragging", "ACE_Explosives", "ACE_Ladder", "ACE_Sandbag", "ACE_refuel", "ACE_rearm", "ACE_dragging"]] call FUNC(statusEffect_addType);
+["forceWalk", false, ["ACE_SwitchUnits", "ACE_Attach", "ACE_dragging", "ACE_Explosives", "ACE_Ladder", "ACE_Sandbag", "ACE_refuel", "ACE_rearm", "ACE_Trenches"]] call FUNC(statusEffect_addType);
```

Comment:
@PabstMirror Is it correct to delete the 2nd `ACE_dragging` here? Honestly I didn't manage to understand `statusEffect` types logic, but from what I understood it seems correct.

Reply:
good catch, they should only be listed once
it's doing `_statusReasons find _ID` to get the index